Dockerfile

```diff
@@ -256,7 +256,8 @@ RUN export COMPOSER_HOME=/composer && \
     mkdir -p /opt/kimai/var/logs && chmod 777 /opt/kimai/var/logs && \
     sed "s/128M/256M/g" /usr/local/etc/php/php.ini-development > /usr/local/etc/php/php.ini && \
     sed "s/128M/-1/g" /usr/local/etc/php/php.ini-development > /opt/kimai/php-cli.ini && \
-    sed -i "s/env php/env -S php -c \/opt\/kimai\/php-cli.ini/g" /opt/kimai/bin/console
+    sed -i "s/env php/env -S php -c \/opt\/kimai\/php-cli.ini/g" /opt/kimai/bin/console && \
```

Comment:
Aren't there directories like `/etc/php/7.3/cli/` and `/etc/php/7.3/fpm/` available, where you could copy this new ini to?